docs/notebooks/gis.ipynb

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
<p><code><strong>The</strong> following watershed sources are currently available: </code> </p><p> </p><p>Aussi, il devrait idéalement y avoir un endroit dans la documentation de <code>xdatasets</code> où les jeux de données disponibles sont listés. Ça faciliterait beaucoup l'utilisation et ça permettrait de simplement référer à cette page de la documentation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>